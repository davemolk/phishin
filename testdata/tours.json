{"success":true,"total_entries":114,"total_pages":6,"page":1,"data":[{"id":1,"name":"1983 Tour","shows_count":1,"slug":"1983-tour","starts_on":"1983-12-02","ends_on":"1983-12-02","shows":[{"id":1324,"date":"1983-12-02","duration":1031524,"incomplete":true,"sbd":true,"remastered":false,"tour_id":1,"venue_id":306,"likes_count":31,"taper_notes":"Phish\r\nHarris-Millis Cafeteria\r\nUniversity Of Vermont\r\nBurlington VT\r\n12/2/83 (portion)\r\n\r\nSource: FM \u003e ?? \u003e CDR \u003e EAC \u003e FLAC\r\nThanks to: Geoff Ecker for the source disc.\r\n\r\nSet II\r\n\r\n1. Scarlet Begonias [9:18] \u003e\r\n2. Fire On The Mountain [7:52]\r\n\r\n\r\nNotes: This is a portion of the first Phish gig that was broadcast on\r\n       \"The Bunny\" Coventry radio broadcast.","updated_at":"2018-12-21T08:10:28Z","venue_name":"Harris-Millis Cafeteria, University of Vermont","location":"Burlington, VT"}],"updated_at":"2013-10-08T04:34:44Z"},{"id":2,"name":"1984 Tour","shows_count":2,"slug":"1984-tour","starts_on":"1984-11-03","ends_on":"1984-12-01","shows":[{"id":1334,"date":"1984-11-03","duration":4214569,"incomplete":true,"sbd":false,"remastered":false,"tour_id":2,"venue_id":610,"likes_count":17,"taper_notes":"Phish\r\n11/3/83\r\nSlade Hall Basement, University of Vermont - Burlington, VT\r\n\r\nSource: Aud \u003e cass/2 \u003e DAT\r\nConversion: DA-P1 \u003e Fiji \u003e Sound Forge 4.5 \u003e CD Wave \u003e SHN v3\r\nConversion and Encoding by Mike Wren \u003cmikew@etree.org\u003e 11/3/00\r\n\r\nThanks Shaggy for the seed!\r\n\r\n--------------------------------------------------------------\r\nhttp://etree.org - The standard in lossless audio distribution\r\n--------------------------------------------------------------\r\n\r\n01  Ignition Sequence\t\t\t\t\t(00:30)\r\n02  The Midnight Hour (Wilson Pickett)\t\t\t(06:22)\r\n03  Wild Child (Lou Reed)\t\t\t\t(05:20)\r\n04  St. Stephen Jam \u003e Bertha (Grateful Dead)\t\t(13:39)\r\n05  Can't You Hear Me Knockin' (Rolling Stones) *\t(09:39)\r\n06  Camel Walk\t\t\t\t\t\t(01:17)\r\n07  Eyes of the World \u003e (Grateful Dead)\t\t\t(17:54)\r\n08  Whipping Post (Allman Brothers)\t\t\t(16:03)\r\n\r\n*With \"St. Stephen\" (Grateful Dead) jam. \r\n\r\nTotal Time: 70:44","updated_at":"2018-12-21T08:10:28Z","venue_name":"Slade Hall, University of Vermont","location":"Burlington, VT"},{"id":2,"date":"1984-12-01","duration":5726850,"incomplete":false,"sbd":true,"remastered":false,"tour_id":2,"venue_id":467,"likes_count":26,"taper_notes":"---------------------------------------------------------------------------\r\ninfo file for version circulated on 12/01/07 (more remastering and 44.1 kHz seed)\r\n---------------------------------------------------------------------------\r\n\r\nPhish - 12/1/84 (w/ full 10/17/85 filler on disc two)\r\nNectar's, Burlington, VT\r\n\r\nThe First Circulated Phish Show\r\n\r\nSource: SBD \u003e Cass1 \u003e DAT \u003e CD-R \u003e EAC \u003e WAV \u003e\r\nCool Edit for Re-mastering (see notes at bottom) \u003e SHN\u003eWAV\u003eFinal Cut Pro for pitch correction\u003eSHN\r\n\r\nRe-mastering, Shortened, md5sums \u0026 uploaded by Mike Wren - mikew@etree.org\r\n\r\nReleased to commemorate the one year anniversary of etree.org\r\n\r\n\r\nPitch correction by Hunter Seamons using Final Cut Pro (12/31/06; remastered 12/1/07)\r\n\r\n\r\n-----\r\n\r\n12/1/84\r\n\r\nDisc 1:\r\n\r\n1) Scarlet Begonias \u003e\r\n2) Fire \u003e\r\n3) Fire on the Mountain\r\n4) Makisupa Policeman\r\n5) Slave to the Traffic Light\r\n6) Spanish Flea\r\n7) Don't Want You No More \u003e\r\n8) Cities \u003e\r\n9) Drums \u003e\r\n10) Skippy the Wondermouse \u003e\r\n\r\n\r\nDisc 2:\r\n\r\n1) Fluffhead\r\n\r\nEncore:\r\n2) Eyes of the World\r\n\r\n-----\r\n\r\nFiller:\r\n\r\n10/17/85\r\n\r\n3) Star Trek Jam \u003e\r\n4) Alumni Blues \u003e\r\n5) Mike's Song\r\n6) Dave's Energy Guide\r\n7) Revolution \u003e\r\n8) Anarchy\r\n9) Camel Walk\r\n10) Run Like an Antelope\r\n11) McGrupp and the Watchful Hosemasters\r\n\r\n\r\n-----\r\n\r\nPREVIOUS NOTES by Mike Wren:\r\n\r\n* Thanks to Lee Farber for the original seed!\r\n\r\n* There was some re-mastering done to clean up the original\r\ntape hiss from this classic show. The processes I took are\r\nas follows:\r\n\r\n1) Amplified both discs 175% across the board.\r\n2) Sampled a section of silence to obtain a decent noise floor\r\nsample (25,000 sample points used).\r\n3) Applied this sample against both discs at 64 percent.\r\n4) Applied a graphic equalizer to boost the high and low spectrums\r\n(specifically between 600 - 800 Hz and 8k \u0026 10 kHz).\r\n***Note: I cut everything above 11 kHz 18dB because nothing\r\nexisted above that freq. range on the original recording.\r\n\r\n\r\nNEW NOTES by Hunter Seamons (12/31/06):\r\n\r\nI noticed the pitch was too high in both shows listed in this seed. I decreased the pitch in 12.01.84 by 7%, and I decreased the pitch in 10.17.85 by 9%.\r\n\r\n\r\nREMASTERED (12/01/07):\r\n\r\nAs this was my first seed, I naturally made some mistakes the first time around. I made all changes by going back to the original file set by Mike Wren and redoing the process, so this is as fresh as it gets. The new changes from my previous seed are as follows:\r\n\r\n1) SBE fix\r\n\r\n2) Encoded at 44.1 kHz (The last source I did was accidently encoded at a \"false\" 48 kHz. Whether this makes any difference sonically has not been determined.)\r\n\r\n3) Different speed reductions at 6% and 8%, respectively. (I believe I was a hair off the first time around, so I wanted to use this opportunity to make the changes.)\r\n\r\n4) NO CUTS! (I made some unorthodox, millisecond cuts before between songs, due to my inexperience of pitch correcting at the time. These went largely unnoticed, as far as I can tell.)\r\n\r\nThese changes should make this source better for the long haul.\r\n\r\nEnjoy this historic show on its 23rd anniversary!","updated_at":"2018-12-21T08:10:12Z","venue_name":"Nectar's","location":"Burlington, VT"}],"updated_at":"2013-11-26T00:03:57Z"}]}