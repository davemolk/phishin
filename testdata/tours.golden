Name:      Starts On:  Ends On:    Shows Count:
1983 Tour  1983-12-02  1983-12-02  1
1984 Tour  1984-11-03  1984-12-01  2
